.checkstyle/suppressions.xml

```diff
@@ -57,4 +57,6 @@
     <suppress checks=".*"
               files="io[/\\]strimzi[/\\]api[/\\]kafka[/\\]model[/\\](Doneable).*\.java"/>
 
+    <suppress checks="NPathComplexity|CyclomaticComplexity"
```

Comment:
Can you please use the annotation in the class it self instead of addint it here?

Reply:
I resorted to this technique because nothing I tried in the class itself worked :/

Reply:
Have you tried `@SuppressWarnings({"checkstyle:CyclomaticComplexity", "checkstyle:NPathComplexity"})`? It is used in other classes already.

Reply:
I'd swear I had tried it but, this time it appears to work. Thanks.